app/src/main/java/com/owncloud/android/ui/fragment/OCFileListFragment.java

```diff
@@ -426,13 +429,16 @@ public void onActivityCreated(Bundle savedInstanceState) {
         setTitle();
 
         FragmentActivity fragmentActivity;
-        if ((fragmentActivity = getActivity()) != null && fragmentActivity instanceof FileDisplayActivity) {
-            FileDisplayActivity fileDisplayActivity = (FileDisplayActivity) fragmentActivity;
+        if ((fragmentActivity = getActivity()) != null && fragmentActivity instanceof FileDisplayActivity fileDisplayActivity) {
```

Comment:
```suggestion
        if (getActivity() instanceof FileDisplayActivity fileDisplayActivity) {
```

Please also remove the previous line.